weeder.dhall

```diff
@@ -1,6 +1,8 @@
 { roots =
   [ "^Main.main\$"
   , "^Paths_"
+  , "^Ormolu.Fixity.defaultFixityMap\$"
```

Comment:
- The weeder bug https://github.com/ocharles/weeder/issues/92 is actually no longer being triggered since #841 as I inlined `hackageInfoFile` (so `"^Ormolu.Fixity.hackageInfoFile\$"` from `weeder.dhall`).
 -  `"^Ormolu.Fixity.defaultFixityMap\$"` is used in Ormolu Live, but we don't build it with GHC (only with GHCJS) on CI, so weeder can't know that it is used there. This could be easily fixed by inlining `defaultFixityMap`, but Ormolu Live's behavior w.r.t. the new fixity stuff will probably be changed either way (e.g. by adding configurability in the interface), so we can do this then.   
 - `type-class-roots = True` is necessary due to a [known limitation](https://github.com/ocharles/weeder#calling-weeder) of weeder which is triggered by `showFixityDirection` in `ToJSON FixityInfo`.

Reply:
I think that we should remove `showFixityDirection` now that `FixityDirection` implements `Show`?